### PR TITLE
URLS are in array, email field is now email type

### PR DESCRIPTION
**Pull request by lbiester**

### CODE REVIEW

File: views/form.jade [hunk 1]
```diff
@@ -21,7 +21,7 @@ mixin userform(user)
     .form-group
       label.control-label.col-sm-4(for='user-urls') Online Profiles (GitHub, LinkedIn, etc.)
       .col-sm-8
-        textarea.form-control(id='user-urls',name='user[urls]', value=(('urls' in user) ? user.classYear : undefined))
+        textarea.form-control(id='user-urls',name='user[urls]', placeholder='urls, one per line' value=(('urls' in user) ? user.classYear : undefined))
```
Comment (Machyne) [inline views/form.jade:24]: Text area default value should be inside the tag instead of in `value="..."`

File: routes.js [hunk 2]
```diff
@@ -24,6 +24,14 @@ var saveUser = function (model, data, callback) {
     dietary.push(key);
   };
   data.dietary = dietary;
+
+  var urls = data.urls.split('\n');
+
+  for (var i = 0; i < urls.length; i++) {
+    urls[i] = urls[i].trim();
+    urls[i] = urls[i].replace(',', '');
```
Comment (Machyne) [inline routes.js:32]: Can you use `validator` (included in the new version of this file) to ensure that these are valid urls?

**Comment by Machyne** [comment]
There are quite a few changes between your commit and `master`. Could you rebase so the auto-merge will work?

**Comment by lbiester** [comment]
I've made all of these changes, can you send me the .api_keys.js so that I can actually get it to run to test?

**Comment by lbiester** [comment]
Okay, I ended up making a new branch that can be merged (form/urls) because the other branch was really screwed up for some reason. I think I filled the form correctly if there are currently urls for the user, but I'm not 100% sure because I don't know jade very well.
Should I PR with the new branch?

**Comment by Machyne** [comment]
I made a few small changes. Feel free to comment, but if it looks good to you, I'll merge. Thanks for the great work!

**Comment by lbiester** [comment]
Cool, looks good from what I saw! I don't really know jade (as I said), thanks for fixing it!

**Pull request closed without merging**